code/src/hooks/use-input-visibility/useInputVisibility.test.ts

```diff
@@ -0,0 +1,25 @@
+import { act, renderHook } from "@testing-library/react";
+import useInputVisibility from "@/hooks/use-input-visibility/useInputVisibility";
+
+describe("useInputVisibility custom hook", () => {
+  test("should use input visibility", () => {
+    const { result } = renderHook(() => useInputVisibility());
+
+    expect(result.current.showInputText).toBe(false);
+    expect(typeof result.current.inputVisibility).toBe("object");
```

Comment:
Why we need this check? I hope it would be better to check for some field in return object, because checking for object is pretty generic

Reply:
It redundent. Changed. Thank you!